cloud/deployment/workerqueue/workerqueue.go

```diff
@@ -35,264 +35,329 @@ var (
 	if err != nil {
 		return err
 	}
 	// change action to past tense
 	succeededAction = fmt.Sprintf("%sd", action)
 
-	fmt.Fprintf(out, "worker queue %s for %s in %s workspace %s\n", queueToCreateOrUpdate.Name, requestedDeployment.Label, ws, succeededAction)
+	fmt.Fprintf(out, "worker queue %s for %s in %s workspace %s\n", name, requestedDeployment.Name, ws, succeededAction)
 	return nil
 }
 
 // SetWorkerQueueValues sets default values for MinWorkerCount, MaxWorkerCount and WorkerConcurrency if none were requested.
-func SetWorkerQueueValues(wQueueMin, wQueueMax, wQueueConcurrency int, workerQueueToCreate *astro.WorkerQueue, workerQueueDefaultOptions astro.WorkerQueueDefaultOptions) *astro.WorkerQueue {
+func SetWorkerQueueValues(wQueueMin, wQueueMax, wQueueConcurrency int, workerQueueToCreate *astroplatformcore.WorkerQueueRequest, workerQueueDefaultOptions astroplatformcore.WorkerQueueOptions) *astroplatformcore.WorkerQueueRequest {
 	// -1 is the CLI default to allow users to request wQueueMin=0
 	if wQueueMin == -1 {
 		// set default value as user input did not have it
-		workerQueueToCreate.MinWorkerCount = workerQueueDefaultOptions.MinWorkerCount.Default
+		workerQueueToCreate.MinWorkerCount = int(workerQueueDefaultOptions.MinWorkers.Default)
 	}
 
 	if wQueueMax == 0 {
 		// set default value as user input did not have it
-		workerQueueToCreate.MaxWorkerCount = workerQueueDefaultOptions.MaxWorkerCount.Default
+		workerQueueToCreate.MaxWorkerCount = int(workerQueueDefaultOptions.MaxWorkers.Default)
 	}
 
 	if wQueueConcurrency == 0 {
 		// set default value as user input did not have it
-		workerQueueToCreate.WorkerConcurrency = workerQueueDefaultOptions.WorkerConcurrency.Default
+		workerQueueToCreate.WorkerConcurrency = int(workerQueueDefaultOptions.WorkerConcurrency.Default)
 	}
 	return workerQueueToCreate
 }
 
-// GetWorkerQueueDefaultOptions calls the workerqueues query
-// It returns WorkerQueueDefaultOptions if the query succeeds
-// An error is returned if it fails
-func GetWorkerQueueDefaultOptions(client astro.Client) (astro.WorkerQueueDefaultOptions, error) {
-	var (
-		workerQueueDefaultOptions astro.WorkerQueueDefaultOptions
-		err                       error
-	)
-	workerQueueDefaultOptions, err = client.GetWorkerQueueOptions()
-	if err != nil {
-		return astro.WorkerQueueDefaultOptions{}, fmt.Errorf("%w: %s", errWorkerQueueDefaultOptions, err.Error())
+// SetWorkerQueueValues sets default values for MinWorkerCount, MaxWorkerCount and WorkerConcurrency if none were requested.
+func SetWorkerQueueValuesHybrid(wQueueMin, wQueueMax, wQueueConcurrency int, workerQueueToCreate *astroplatformcore.HybridWorkerQueueRequest, workerQueueDefaultOptions astroplatformcore.WorkerQueueOptions) *astroplatformcore.HybridWorkerQueueRequest {
+	// -1 is the CLI default to allow users to request wQueueMin=0
```

Comment:
to match code
```suggestion
	// -1 is the CLI default to allow users to request wQueueMin=default
```

Reply:
I don't think 0 is default? what does this mean?

Reply:
the comment says `// -1 is the CLI default to allow users to request wQueueMin=0`
I think it should say
```suggestion
	// -1 is the CLI default to allow users to request wQueueMin=default
```

Reply:
okat